Makefile

```diff
@@ -29,11 +28,11 @@ dev_setup: $(VENV)/bin/activate	 ## Install all dependencies and setup pre-commi
 	make install_deps
 	$(VENV)/bin/pre-commit install
 
-functional_test:	## Run functional tests.
-	$(VENV)/bin/python3 -m pytest --profile $(PROFILE) $(TESTS) $(FLAGS)
+test: ## Test specific version of python
+	tox --recreate -e $(PYTHON_VERSION) -- $(TESTS) --profile $(PROFILE)
```

Comment:
tox won't be able as a top-level command in the shell. Either add it to the shell or provide the path to tox. 

```
make test TESTS=tests/functional/adapter/test_basic.py::TestSimpleMaterializationsHive PYTHON_VERSION=py38
tox --recreate -e py38 -- tests/functional/adapter/test_basic.py::TestSimpleMaterializationsHive --profile dwx_endpoint
make: tox: Command not found
make: *** [Makefile:32: test] Error 127

```

Reply:
Thanks for catching this. Updated the PR.